src/components/Counter/Counter.jsx

```diff
@@ -7,39 +8,61 @@ const colors = {
   red: 'rgb(239, 68, 68)',
 };
 
+const initialCount = { count: 0, color: colors.green };
```

Comment:
Since our initial count is placed at zero, we will want the color to be colors.yellow. I believe the useEffect is overriding the initial variable, just something to watch out for in the future.